src/app/components/main/create-superhero/create-superhero.component.html

```diff
@@ -0,0 +1,32 @@
+<div [ngClass]="newSuperhero.affiliation" class="create-superhero-container">
```

Comment:
This is outdated and going to be entirely reworked. So it should just be removed/emptied